doc/deployment/deploy_custom.md

```diff
@@ -320,41 +295,43 @@ The Pachyderm's service account is called  `pachyderm`.
 
 ### Role or ClusterRole
 
-Depending on whether you used the `--local-roles` flag or not, the next manifest will be of `kind` `Role` or `ClusterRole`.
-depending on whether you  used the `--local-roles` flag `pachctl deploy` command.
+Depending on whether you used the `--local-roles` flag or not, the next manifest `kind` is `Role` or `ClusterRole`.
 
 ### RoleBinding or ClusterRoleBinding
 
-This manifest binds the `Rule` or `ClusterRole` to the `ServiceAccount` created above.
+This manifest binds the `Role` or `ClusterRole` to the `ServiceAccount` created above.
 
 ## Application-related
 
+This section walks you through the application-related manifests.
+
 ### PersistentVolume
 
 If you used `--static-etc-volume` to deploy Pachyderm,
 the value that you specify for `--persistent-disk` causes `pachctl` to write a manifest for creating a [`PersistentVolume`](https://kubernetes.io/docs/concepts/storage/persistent-volumes/) that Pachyderm's `etcd` uses in its [`PersistentVolumeClaim`](https://kubernetes.io/docs/concepts/storage/persistent-volumes/#persistentvolumeclaims).
 
-A common persistent volume uses in enterprises is an NFS mount backed by a storage fabric of some sort.
-In this case, a StorageClass for an NFS mount will be made available for consumption.
-Consult with your Kubernetes administrators to learn what resources are available for your deployment
+A common persistent volume that is used in enterprises is an NFS mount backed by a storage fabric.
+In this case, a StorageClass for an NFS mount is made available for consumption.
+Consult with your Kubernetes administrators to learn what resources are available for your deployment.
 
 ### PersistentVolumeClaim
 
-Pachyderm's `etcd` uses this `PersistentVolumeClaim` if you deployed using `--static-etc-volume`.
+If you deployed Pachyderm by using `--static-etc-volume`, the Pachyderm's `etcd` uses this `PersistentVolumeClaim`.
 See this manifest's name in the `etcd` Deployment manifest below.
 
 ### StorageClass
 
 If you used the `--dynamic-etcd-nodes` flag to deploy Pachyderm,
 this manifest specifies the kind of storage and provisioner that is appropriate for what you have specified in the `--persistent-disk` flag. 
 
-(**NOTE** You won't see this manifest if you specified `azure` as the argument to `--persistent-disk`.)
+**Note:** You will not see this manifest if you specified `azure` as the argument to `--persistent-disk`.
 
 ### Service
 
-In a typical Pachyderm deployment, you see three [`Service`](https://kubernetes.io/docs/concepts/services-networking/service/) manifests. 
+In a typical Pachyderm deployment, 
+you see three [`Service`](https://kubernetes.io/docs/concepts/services-networking/service/) manifests. 
 Services are how Kubernetes exposes Pods to the network.
-If you  use StatefulSets to deploy Pachyderm,
+If you use StatefulSets to deploy Pachyderm,
```

Comment:
> To make your `containerPort` values work properly, add environment variables to a `Deployment` or `StatefulSet` object.
You can verify this functionality in the [OpenShift](./openshift.html) example.	You can verify this functionality in the [OpenShift](./openshift.html) example.

Does one _need_ to add environment variables to make this work properly? What does "work properly" mean? Is this only required for openshift or always? Make this sentence clearer. If it's a require set of steps, it's pretty darn hidden away in this document. If it's "you can customize you deployment with these envvars" then thats fine, but still should clarify what these do and when they should be used.